packages/api-plugin/src/generators/trpc-api-generator/files/tsup.config.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
```

Comment:
it might be good to include `dts: true` in this so it will output `*.d.ts` files

Reply:
good point